### PR TITLE
Master

### PR DESCRIPTION
**Pull request by AndrewJWashington**
Issue #44

### CODE REVIEW

File: neurodsp/burst.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 
-def detect_bursts(Fs, x, f_range, algorithm, thresh, magnitudetype='amplitude',
+def detect_bursts(x, Fs, f_range, algorithm, thresh, magnitudetype='amplitude',
```
Comment (srcole) [inline neurodsp/burst.py]: can you also change the ordering of these parameters in the docstring

File: neurodsp/burst.py [hunk 2]
```diff
@@ -96,6 +96,51 @@ def detect_bursts(Fs, x, f_range, algorithm, thresh, magnitudetype='amplitude',
         return isosc_noshort
 
 
+def get_stats(bursting, Fs):
+    """
+    Get statistics of bursts.
+
+    Parameters
+    ----------
+    x : array-like 1d
```
Comment (srcole) [inline neurodsp/burst.py]: 'bursting'?

Comment (AndrewJWashington) [reply neurodsp/burst.py]: Fixed

File: neurodsp/burst.py (hunk 1 above)
Comment (AndrewJWashington) [reply neurodsp/burst.py]: Done

**Comment by AndrewJWashington** [comment]
Ready to merge!

**Comment by srcole** [comment]
Tutorial adds new data to satisfy issue #43

**Pull request merged**